ui/pages/settings/flask/snaps-list-tab/snap-list-tab.component.js

```diff
@@ -51,9 +51,9 @@ const SnapListTab = ({
     );
   }
   return (
-    <>
+    <div className="snap-list-tab">
```

Comment:
Nit: This was fine beforehand - the `snap-list-tab` class doesn't have any rules in it, so it doesn't need to be set anywhere.